caproto/_dbr.py

```diff
@@ -149,6 +149,12 @@ class _LongStringChannelType(IntEnum):
     CTRL_LONG_STRING = 32
 
 
+_channel_type_by_name = {
```

Comment:
Does this get used someplace else?

Reply:
Just by the stuff in this PR, but could presumably be used by other stuff.